x/staking/proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -25,16 +25,6 @@ message HistoricalInfo {
   repeated Validator       valset = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 }
 
-// HistoricalRecord contains a set of minimum values needed for evaluating historical validator sets and blocks.
```

Comment:
Shouldn't we not keep this and mark it as deprecated too?

Reply:
This was introduced in an unreleased version so I opted to remove

Reply:
Oh it's unreleased, then indeed it makes sense